### PR TITLE
feat(files): add missing fields to images and documents

### LINKED ISSUE #193: Add Image Focal point fields
Documentation says we can access focal point fields for an [ImageObjectType](https://wagtail-grapple.readthedocs.io/en/latest/general-usage/graphql-types.html#imageobjecttype). But these fields are not available at the APIs.

Looks like we need to add them at [BaseImageObjectType](https://github.com/GrappleGQL/wagtail-grapple/blob/e92abab64ea0dcacf68cd68ccd64a66c055e55f6/grapple/types/images.py#L17)?

### PR DESCRIPTION
**Pull request by ruisaraiva19**
- Closes #193

### CODE REVIEW

**Review by sks444 — APPROVED** [review]
This works well for me.

Thanks @ruisaraiva19!

File: grapple/types/tags.py [hunk 1]
```diff
@@ -3,13 +3,13 @@
 from graphene_django.converter import convert_django_field
 from grapple.utils import resolve_queryset
 
-from modelcluster.contrib.taggit import ClusterTaggableManager
```
Comment (zerolab) [inline grapple/types/tags.py:6]: question: what prompted this change?

Comment (ruisaraiva19) [reply grapple/types/tags.py:6]: The `ClusterTaggableManager` extends the `TaggableManager` class, so by registering the base `TaggableManager` class we get proper values on the resolvers for all tag fields.

Comment (zerolab) [reply grapple/types/tags.py:6]: 🤔 this'll probably fail on previews as modelcluster allow for that without objects being saved in the database

Comment (ruisaraiva19) [reply grapple/types/tags.py:6]: Hum, that's a good point. Maybe we can register both managers?

Comment (zerolab) [reply grapple/types/tags.py:6]: hmm,

tried 
```
@convert_django_field.register(TaggableManager)
@convert_django_field.register(ClusterTaggableManager)
def convert_tag_manager_to_string(field, registry=None):
    return TagObjectType(
```

locally and it doesn't explode. so let's go with that

Comment (ruisaraiva19) [reply grapple/types/tags.py:6]: Updated!

**Pull request merged**